app/src/main/java/ar/com/wolox/android/cookbook/CookbookApplication.kt

```diff
@@ -41,6 +47,16 @@ class CookbookApplication : WolmoApplication() {
         Fresco.initialize(this)
     }
 
+    private fun initializeTwitter() {
+        val config = TwitterConfig.Builder(this)
+                .logger(DefaultLogger(Log.DEBUG)) // enable logging when app is in debug mode
+                .twitterAuthConfig(TwitterAuthConfig(this.resources.getString(R.string.twitter_consumer_key),
+                        this.resources.getString(R.string.twitter_consumer_secret)))
+                .debug(true) // enable debug mode
```

Comment:
Maybe you can check BuildConfig.DEBUG to enable or disable the logs. This allows us yo change the build variant and get this logs or not (if its a release). What do you think?